src/test/java/com/wikia/webdriver/testcases/admindashboardtests/EditingLocalCssTests.java

```diff
@@ -43,12 +43,12 @@ public void EditingLocalCss_002_UserWithoutAdminRightsHasNoEditOption() {
     WikiBasePageObject base = new WikiBasePageObject(driver);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "base" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)